### ISSUE TITLE
[Issue]: 'removed 3905 packages in 23s' >>> panic: runtime error: slice bounds out of range [:1] with length 0

### ISSUE THREAD

**Issue by PeterPCW** [coreybutler/nvm-windows#1009]
### What happened?

I had an eslint conflict gumming up my build. Updating the local package didn't help so I tried to update globally and it seems to have uninstalled all of npm? Node still gives a version, and nvm can't reinstall like this. Same error as #942.

### What did you expect to happen?

Update eslint. Or in the nvm case to reinstall npm.

### Version

1.1.10

### Which version of Windows?

Windows 10 (Default)

### Which locale?

English (Default)

### Which shell are you running NVM4W in?

Command Prompt

### User Permissions?

Administrative Privileges, Non-Elevated

### Is Developer Mode enabled?

No (Default)

### Relevant log/console output

```Shell
C:\...>npm -g update eslint-config-react-app

removed 3905 packages in 23s

C:\...>npm version
'npm' is not recognized as an internal or external command,
operable program or batch file.

C:\...>nvm install npm
panic: runtime error: slice bounds out of range [:1] with length 0

goroutine 1 [running]:
main.versionNumberFrom({0x1211c078, 0x3})
        C:/Users/corey/OneDrive/Documents/workspace/libraries/oss/coreybutler/nvm-windows/src/nvm.go:496 +0x116
main.getVersion({0x1211c078, 0x3}, {0xdc9d26, 0x2}, {0x0, 0x0, 0x0})
        C:/Users/corey/OneDrive/Documents/workspace/libraries/oss/coreybutler/nvm-windows/src/nvm.go:233 +0x367
main.install({0x1211c078, 0x3}, {0xdc9d26, 0x2})
        C:/Users/corey/OneDrive/Documents/workspace/libraries/oss/coreybutler/nvm-windows/src/nvm.go:273 +0xbb
main.main()
        C:/Users/corey/OneDrive/Documents/workspace/libraries/oss/coreybutler/nvm-windows/src/nvm.go:87 +0xaea

C:\...>node -v
v18.15.0
```

### Anything else?

#942 https://github.com/coreybutler/nvm-windows/issues/942

**Comment by PeterPCW** [comment]
I was on 1.1.10 and the 942 fix was 1.1.11 so I uninstalled nvm and started over at 1.1.11. Seems to be working fine now but I don't know if this is the same issue or not. Feel free to close as duplicate if so.

As an aside I would love to understand what happened with `npm update -g [pkg]` which should be valid?
https://docs.npmjs.com/cli/v6/commands/npm-update

**Comment by coreybutler** [comment]
The one glaring thing I see in the log is `nvm install npm`. NVM4W doesn't have a command like this because it is expecting a Node version or supported alias. Since `npm` is neither, it threw the error. In v1.1.11, these errors are caught and display a proper error message (as shown in 942).

Once npm is installed (during the Node install), NVM4W backs off entirely. Running `npm _anything_` runs npm directly without NVM4W involvement. Bottom line, `npm update -g [pkg]` would be an npm issue or an issue for the specific module you're installing, not an NVM for Windows problem.

Closing because the error is a dupe of 942 and the later functionality (nvm install npm) is not a valid command.

For anyone else struggling with this, here are a few key points.

1. If npm is jacked up, you can always remove it from the node installation manually.
2. You can manually install npm by downloading it from the npm repo and dumping it in the node_modules directory of the desired Node installation.
